pontoon/tour/migrations/0001_squashed_0001_initial.py

```diff
@@ -40,6 +40,7 @@ def create_tutorial_project(apps, schema_editor):
         admin_notes=(
             "Do not remove, this is required in code to show the guided tour."
         ),
+        visibility="public",
```

Comment:
Why is this change needed?

We already made existing projects public in a data migration:
https://github.com/mozilla/pontoon/blob/master/pontoon/base/migrations/0004_make_old_projects_public_again.py

Reply:
On my machine (and on TravisCI too), migration `base.0004`  executes before migration `tour.001`.

However, I've decided to revert this change and add a data-migration to fix it.